README.md

```diff
@@ -107,6 +107,34 @@ print(f"Similarity: {int((y_pred_fhe == y_pred_clear).mean()*100)}%")
     # Similarity: 100%
 ```
 
+It is also possible to call encryption, model prediction, and decryption functions separately as follows.
+Executing these steps separately is equivalent to calling `predict_proba` on the model instance.
+
+<!--pytest-codeblocks:cont-->
+
+```python
+# Predict probability for a single example
+y_proba_fhe = model.predict_proba(X_test[[0]], fhe="execute")
+
+# Quantize an original float input
+q_input = model.quantize_input(X_test[[0]])
+
+# Encrypt the input
+q_input_enc = model.fhe_circuit.encrypt(q_input)
+
+# Execute the linear product in FHE
+q_y_enc = model.fhe_circuit.run(q_input_enc)
+
+# Decrypt the result (integer)
+q_y = model.fhe_circuit.decrypt(q_y_enc)
+
+# De-quantize and post-process the result
+y0 = model.post_processing(model.dequantize_output(q_y))
+
+print("Probability with `predict_proba`: ", y_proba_fhe)
+print("Probability with encrypt/run/decrypt calls: ", y0)
+```
```

Comment:
thx; I would add an assert to be sure it works well

Reply:
I wanted to but here we return floats so it's a bit of a pain to assert on allclose.

Reply:
we have pytests for that

Reply:
pytest will break: you'll fix pytest
but no one will think about fixing the doc
that's why having minimal checks in the doc is better, I think